vision/cloud-client/detect/beta_snippets.py

```diff
@@ -0,0 +1,350 @@
+                print(u'Word text: {} (confidence: {})\n'.format(
+                    word_text, word.confidence))
+
+                block_text += ' ' + word_text
+
+            print(u'Block Content: {}\n'.format(block_text))
+            print(u'Block Confidence:\n {}\n'.format(block.confidence))
+# [END def_detect_document_uri]
+
+
+# [START def_detect_safe_search]
+def detect_safe_search(path):
+    """Detects unsafe features in the file."""
+    client = vision.ImageAnnotatorClient()
+
+    with io.open(path, 'rb') as image_file:
+        content = image_file.read()
+
+    image = vision.types.Image(content=content)
+
+    response = client.safe_search_detection(image=image)
+    safe = response.safe_search_annotation
+
+    # Names of likelihood from google.cloud.vision.enums
+    likelihood_name = ('UNKNOWN', 'VERY_UNLIKELY', 'UNLIKELY', 'POSSIBLE',
+                       'LIKELY', 'VERY_LIKELY')
+    print('Safe search:')
+
+    print('adult: {}'.format(likelihood_name[safe.adult]))
+    print('medical: {}'.format(likelihood_name[safe.medical]))
+    print('spoofed: {}'.format(likelihood_name[safe.spoof]))
+    print('violence: {}'.format(likelihood_name[safe.violence]))
+    print('racy: {}'.format(likelihood_name[safe.racy]))
+# [END def_detect_safe_search]
+
+
+# [START def_detect_safe_search_uri]
+def detect_safe_search_uri(uri):
+    """Detects unsafe features in the file located in Google Cloud Storage or
+    on the Web."""
+    client = vision.ImageAnnotatorClient()
+    image = vision.types.Image()
+    image.source.image_uri = uri
+
+    response = client.safe_search_detection(image=image)
+    safe = response.safe_search_annotation
+
+    # Names of likelihood from google.cloud.vision.enums
+    likelihood_name = ('UNKNOWN', 'VERY_UNLIKELY', 'UNLIKELY', 'POSSIBLE',
+                       'LIKELY', 'VERY_LIKELY')
+    print('Safe search:')
+
+    print('adult: {}'.format(likelihood_name[safe.adult]))
+    print('medical: {}'.format(likelihood_name[safe.medical]))
+    print('spoofed: {}'.format(likelihood_name[safe.spoof]))
+    print('violence: {}'.format(likelihood_name[safe.violence]))
+    print('racy: {}'.format(likelihood_name[safe.racy]))
+# [END def_detect_safe_search_uri]
+
+
+def annotate(path):
+    """Returns web annotations given the path to an image."""
+    # [START get_annotations]
+    client = vision.ImageAnnotatorClient()
+
+    if path.startswith('http') or path.startswith('gs:'):
+        image = vision.types.Image()
+        image.source.image_uri = path
+
+    else:
+        with io.open(path, 'rb') as image_file:
+            content = image_file.read()
+
+        image = vision.types.Image(content=content)
+
+    response = client.web_detection(image=image)
+    web_detection = response.web_detection
+    # [END get_annotations]
+
+    return web_detection
+
+
+def report(annotations):
+    """Prints detected features in the provided web annotations."""
+    # [START print_annotations]
```

Comment:
Can we rename this to something that explicitly calls out this snippet demonstrates Web Detection Using Best Guess Labels? Perhaps something like `web_detection_using_best_guess_labels`?